orc8r/tools/fab/pkg.py

```diff
@@ -123,70 +121,69 @@ def upload_pkgs_to_aws():
     """
 
     # Get the version of magma we are releasing
-    magma_version = get_magma_version()
-    copy_packages()
+    magma_version = get_magma_version(c_agw)
+    copy_packages(c_agw)
 
     # Upload to AWS
     s3_path = 's3://magma-images/gateway/' + magma_version
-    local('aws s3 cp /tmp/packages.txt ' + s3_path + '.deplist')
-    local('aws s3 cp release/magma.lockfile.debian ' + s3_path + '.lockfile.debian')
-    local('aws s3 cp /tmp/packages.tar.gz ' + s3_path + '.deps.tar.gz')
+    c_agw.run('aws s3 cp /tmp/packages.txt ' + s3_path + '.deplist')
+    c_agw.run('aws s3 cp release/magma.lockfile.debian ' + s3_path + '.lockfile.debian')
+    c_agw.run('aws s3 cp /tmp/packages.tar.gz ' + s3_path + '.deps.tar.gz')
 
     # Clean up
-    run('rm -r /tmp/packages')
-    run('rm /tmp/packages.tar.gz')
-    local('rm /tmp/packages.tar.gz')
-    local('rm /tmp/packages.txt')
+    c_agw.run('rm -r /tmp/packages')
+    c_agw.run('rm /tmp/packages.tar.gz')
+    c_agw.run('rm /tmp/packages.tar.gz')
```

Comment:
Why are we running this command twice?

Reply:
Thanks for catching it, it was run on the local machine and the vm before, I changed it to behave the same way again.